src/logic/tokens/store/actions/fetchTokens.js

```diff
@@ -37,25 +39,117 @@ const createERC721TokenContract = async () => {
   return erc721Token
 }
 
+const OnlyBalanceToken = {
```

Comment:
can we move this to a separate file?

Reply:
I'm think it's not a good idea because is an internal method used only in `fetchTokenBalances`

Reply:
it's an ABI definition, does not matter if it's only used here, only generated a bigger file.